From 4e4b27c44bde78cc899af58b719ca3f1abd7fe92 Mon Sep 17 00:00:00 2001
From: Ian Abbott <abbotti@mev.co.uk>
Date: Tue, 14 Apr 2009 11:04:24 -0400
Subject: Staging: comedi: Undo stupid commit made 3 months ago

Undo stupid commit made 3 months ago :
 "Fix redefinition of macro comedi_rt_task_context_t".
 It wasn't being redefined, it was being defined for RTLinux.

From: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/comedi/drivers/comedi_rt_timer.c
+++ b/drivers/staging/comedi/drivers/comedi_rt_timer.c
@@ -104,6 +104,8 @@ static inline RTIME nano2count(long long ns)
 #define start_rt_timer(x)
 #define stop_rt_timer()
 
+#define comedi_rt_task_context_t	int
+
 #endif
 #ifdef CONFIG_COMEDI_RTAI
 #include <rtai.h>
-- 
1.6.2.3

